DataFormats/Phase2TrackerDigi/interface/Phase2TrackerDigi.h

```diff
@@ -35,7 +35,7 @@ class Phase2TrackerDigi {
   unsigned int strip()   const { return row(); }
   unsigned int edge()    const { return column(); } // CD: any better name for that? 
   // Access to the (raw) channel number
-  unsigned int channel() const { return theChannel; }
+  unsigned int channel() const { return 0x7FFF & theChannel; }
```

Comment:
is packedPosition() then also buggy? Strange to have the same thing returned by two functions in an interface.